src/Transport.ts

```diff
@@ -331,320 +338,383 @@ export default class Transport {
   async request<TResponse = unknown, TContext = any> (params: TransportRequestParams, options?: TransportRequestOptionsWithMeta): Promise<TransportResult<TResponse, TContext>>
   async request<TResponse = unknown> (params: TransportRequestParams, options?: TransportRequestOptions): Promise<TResponse>
   async request (params: TransportRequestParams, options: TransportRequestOptions = {}): Promise<any> {
-    const connectionParams: ConnectionRequestParams = {
-      method: params.method,
-      path: params.path
-    }
+    const otelTracer = opentelemetry.trace.getTracer('elastic-transport')
```

Comment:
At least in the OTel repos the instrumentation scope name and version (the arguments to `getTracer()`) are typically the npm package name and version. So perhaps:

```suggestion
    const otelTracer = opentelemetry.trace.getTracer('@elastic/transport', clientVersion)
```